msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/msalonly/ltw/TestCase2571361.java

```diff
@@ -20,13 +20,13 @@
 //  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 //  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 //  THE SOFTWARE.
-package com.microsoft.identity.client.msal.automationapp.testpass.msalonly.basic;
-
-import androidx.test.uiautomator.UiObject;
+package com.microsoft.identity.client.msal.automationapp.testpass.msalonly.ltw;
```

Comment:
Where are these tests in the msalonly package? Seems like all of these are brokered tests, no?

Reply:
These tests should go in `.broker.ltw`, not `.msalonly.ltw`

Reply:
+1. These should be under broker folder